charts/ratify/templates/ratify-manager-role-role.yaml

```diff
@@ -0,0 +1,24 @@
+{{- if .Values.rbac.create }}
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  creationTimestamp: null
+  name: ratify-manager-role
+rules:
+- apiGroups:
+  - ""
+  resources:
+  - serviceaccounts
```

Comment:
actually we need this to be part of a cluster role to enumerate all service accounts across namespaces to query for image pull secrets. Ratify technically does support this use case

Reply:
gotcha, thanks for reminder!

Reply:
Hi @akashsinghal just have some questions on the service accounts details
from doc
https://github.com/deislabs/ratify/blob/main/docs/reference/oras-auth-provider.md#3-kubernetes-secrets
it seems the use case is to let customer patch imagePullSecret instead of reading service accounts accross namespaces.
Besides, the store crd seems also not supporting inputing multiple service account from different namespace.

Is the document out of update that missing the use case?

Reply:
You're right. Apologies for confusion. It's only for secrets in different namespaces. not service account. cc: @binbin-li